dev_guide/getting_traffic_into_cluster.adoc

```diff
@@ -94,6 +94,21 @@ When a set of routes is created in various projects, the overall set of routes
 is available to the set of routers. Each router admits (or selects) routes from
 the set of routes. By default, all routers admit all routes.
 
+As of {product-title} 3.6, routers can accept IPV6 traffic from outside the cluster for the following routes:
+
+* Route type 1
+* Route type 2
+
```

Comment:
@lihongan Sorry, this is just a placeholder for the route types. I know not all are accepted, but I'm not sure which types. I've contacted @imcsk8 for more information. However, if you know which route types are able to be accepted, please let me know.